com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -2,99 +2,101 @@
 
 [High Dynamic Range](https://en.wikipedia.org/wiki/High-dynamic-range_imaging) content has a wider color gamut and greater luminosity range than standard definition content.
```

Comment:
I made a mistake here and gave you the wrong link. The right link is https://en.wikipedia.org/wiki/High_dynamic_range

Reply:
@FrancescoC-unity  just checking in about this change.